database-mongodb/generator.js

```diff
@@ -2,7 +2,6 @@ const faker = require('faker');
 const fs = require('fs');
 const csvWriter = require('csv-write-stream');
 const zlib = require('zlib');
-const gzip = zlib.createGzip();
```

Comment:
Is this instead of draining?

Reply:
oh no this actually I need to remove since i'm no longer using gzip compression.  my draining takes place on lines 37-39 of this file.